lib/vmpooler/api/v1.rb

```diff
@@ -45,35 +45,39 @@ def alias_deref(hash)
       newhash
     end
 
-    def checkout_vm(template, result)
-      vm = backend.spop('vmpooler__ready__' + template)
+    def fetch_single_vm(template)
+      backend.spop('vmpooler__ready__' + template)
```

Comment:
If it does, it shouldn't happen any more often than before -- we effectively factored this out and changed the order of operations, but this isn't net new calls to spop.